docs/articles/documentation/using-testcafe/programming-interface/Global.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+title: Global Members
+permalink: /documentation/using-testcafe/programming-interface/global/
+---
+# Global Members
+
+## Functions
+
+### <a class="anchor" name="createTestCafe"></a>createTestCafe(hostname, port1, port2) → Promise\<[TestCafe](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/)\>
+
+Creates a [TestCafe](/testcafe/documentation/using-testcafe/programming-interface/TestCafe/) server instance.
+
+Parameter  | Type   | Description
+---------- | ------ | ----------------------------------------------------------------------------------------------------
+`hostname` | String | The hostname or IP you will use to address the TestCafe server. Must resolve to the current machine.
+`port1`    | Number | The port to which the TestCafe server will listen.
+`port2`    | Number | The port that will be reserved for internal needs.
```

Comment:
> The port that will be reserved for internal needs.

That's not true. Both ports are used by TestCafe to server testable pages.

We need to mention that all args are optional, 
Regarding `hostname`: if it's not specified TestCafe will try to use one of the following (in that order, until it will not resolved to the current machine):
1. OS hostname
2. Network IP address of the current machine.

Also, if you want to test external devices you need to specify hostname via which your machine is visible in the network shared with that device.